From 01de4d9b0511490da7d9e372d1cb301be7c4cc38 Mon Sep 17 00:00:00 2001
From: gouarin <loic.gouarin@gmail.com>
Date: Thu, 14 Dec 2017 06:09:19 +0100
Subject: [PATCH] fix static cppzmq

---
 CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 48d0f3e..6626596 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,7 +13,6 @@ project(xeus)
 # ============
 
 find_package(xtl REQUIRED)
-find_package(ZeroMQ QUIET)
 find_package(cppzmq REQUIRED)
 find_package(cryptopp REQUIRED)
 
@@ -144,7 +143,7 @@ message(STATUS "${PROJECT_NAME} version: v${${PROJECT_NAME}_VERSION}")
 # ======
 
 set(XEUS_TARGET xeus)
-set(XEUS_DEPENDENCIES ${cppzmq_LIBRARY} cryptopp-static)
+set(XEUS_DEPENDENCIES ${cppzmq_STATIC_LIBRARY} cryptopp-static)
 
 if(NOT MSVC)
     if(NOT APPLE)
-- 
2.14.1

