From ae4a7b5db21203e3de133838b50f25787069f74a Mon Sep 17 00:00:00 2001
From: gouarin <loic.gouarin@gmail.com>
Date: Thu, 14 Dec 2017 06:46:38 +0100
Subject: [PATCH] remove shared lib

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6626596..17c44a7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -152,7 +152,7 @@ if(NOT MSVC)
     endif()
 endif()
 
-add_library(${XEUS_TARGET} SHARED ${XEUS_SOURCES} ${XEUS_HEADERS})
+add_library(${XEUS_TARGET} ${XEUS_SOURCES} ${XEUS_HEADERS})
 target_link_libraries(${XEUS_TARGET} ${XEUS_DEPENDENCIES})
 set_target_properties(${XEUS_TARGET} PROPERTIES
                       PUBLIC_HEADER "${XEUS_HEADERS}"
-- 
2.14.1

